packages/fuselage-hooks/src/useClipboard.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+
+import { useMutableCallback } from './useMutableCallback';
+
+/**
+ * Hook to copy the passed content to the clipboard.
+ *
+ * @returns an object with the copy function and the hasCopied state
+ * @public
+ */
+
+type UseClipboardParams = {
+  clearTime?: number;
+  onCopySuccess?: (e?: Event) => void;
+  onCopyError?: (e?: Event) => void;
+};
+
+export type UseClipboardReturn = {
+  copy: (e?: Event) => Promise<void>;
+  hasCopied: boolean;
+};
+
+export const useClipboard = (
```

Comment:
```suggestion
type UseClipboardParams = {
  clearTime?: number;
  onCopySuccess?: (e?: Event) => void;
  onCopyError?: (e?: Event) => void;
};

export type UseClipboardReturn = {
  copy: (e?: Event) => Promise<void>;
  hasCopied: boolean;
};

/**
 * Hook to copy the passed content to the clipboard.
 *
 * @returns an object with the copy function and the hasCopied state
 * @public
 */
export const useClipboard = (
```

Doc comments have the inconvenient of requiring to be placed above the declaration they refer, without blank lines between them.

Reply:
Nice to know about. I already fixed it